app/assets/stylesheets/custom.css.scss

```diff
@@ -8,16 +20,28 @@ body {
 .header {
   padding-left: 15px;
   padding-right: 15px;
+  padding-bottom: 19px;
   margin-bottom: 15px;
   border-bottom: 1px solid #e5e5e5;
+  h3 {
+    padding-bottom: 0px;
+    margin-top: 0;
+    margin-bottom: 0;
+    line-height: 40px;
+  }
 }
 
-// Make the masthead heading the same height as the navigation
-.header h3 {
-  padding-bottom: 19px;
-  margin-top: 0;
-  margin-bottom: 0;
-  line-height: 40px;
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 60px;
+  background-color: #f5f5f5;

```

Comment:
Color literals like `#f5f5f5` should only be used in variable declarations; they should be referred to via variable everywhere else.